runtime/gen/main.go

```diff
@@ -100,6 +103,7 @@ func starlarkWidgetFromRenderWidget(w render.Widget) *StarlarkWidget {
 	typ := val.Type()
 
 	sw.Name = typ.Name()
+	sw.FullName = typ.String()
```

Comment:
This returns strings like `animation.AnimatedPositioned` or `render.Text`. Not sure if there is a better way to get this string.